server/tsconfig.json

```diff
@@ -21,5 +21,9 @@
     "noUnusedLocals": true,
     "plugins": [],
     "moduleResolution": "node",
-  }
+  },
+	"exclude": [
```

Comment:
Let's not separate the build process, seems complicated.

Reply:
The merged graphql isn't generated until we build `typeDefs.ts`, and we can't get the appropriate types (such as `PluginMaster`) for successful building of `app.ts` without merged graphql types